framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -986,7 +1039,8 @@ private void handleExpr(AnnotationBuilder builder, String name,
                     builder.setValue(name, arr);
                 }
             } else {
-                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr + " and expected: " + expected);
+                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr +
+                        " and expected: " + expected);
```

Comment:
If the operators are all hidden at the end of the line, then I don't know what happens next.  If they are nicely lined up at the beginning, then I can tell.  I guess your preference is the opposite of mine.

Sun's style guide says to put the break after the operator.  The Google style guide, the Mozilla style guide, and Checkstyle say to put the break before the operator.  I agree with the rationale behind the latter style guides.  But there is no general consensus.

You could also leave the expressions on a single line and not break them, and then we don't have to have the discussion. :-)